server/src/Security/Factory/UuidV6Factory.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Security\Factory;
+
+use Symfony\Component\Uid\Uuid;
+use Symfony\Component\Uid\UuidV6;
+
+class UuidV6Factory
```

Comment:
Un petite interface pour respecter l'inversion de dépendance, et ce serait top

Reply:
to be corrected indeed

Reply:
a priori c'est corrigé